apps/studio/components/interfaces/Organization/BillingSettings/PaymentMethods/PaymentMethods.tsx

```diff
@@ -98,16 +99,14 @@ const PaymentMethods = () => {
                       variant="info"
                       title="Payment is currently by invoice"
                       actions={[
-                        <Button key="payment-method-support" asChild type="default">
-                          <Link
-                            href={`/support/new?category=billing&subject=Request%20to%20change%20payment%20method`}
-                            target="_blank"
-                            rel="noreferrer"
-                            className="ml-3"
-                          >
-                            Contact support
-                          </Link>
-                        </Button>,
+                        <ContactSupportButton
+                          category="BILLING"
+                          subject="Request to change payment method"
+                          message="I would like to change my payment method"
+                          errorContext={{
+                            organizationSlug: slug,
+                          }}
```

Comment:
hmm likewise as in CurrentPaymentMethod.tsx - don't think this needs an error context since its not an error